src/Sdk.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Sentry;
+
+use Sentry\Breadcrumbs\Breadcrumb;
+use Sentry\State\Hub;
+
+function init(array $options = []): void
```

Comment:
We don't really need it since all the function in SDK use `Hub::getCurrent()`. And you don't really need the `Hub` expect you are doing something special if so you should use `Hub::getCurrent()` instead of keeping a reference to the hub that `init` might return.